src/components/Mascot-error.svelte

```diff
@@ -0,0 +1,16 @@
+<script>
```

Comment:
I would rather extend the existing `<Mascot />` component to accomodate this, by adding a prop. For example

`export let image`

and it could be either `"jetpack"` or `"error"`.